lib/resources/r4/document_reference.yaml

```diff
@@ -29,3 +116,143 @@ fields:
+        }
+      ]
+    }
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Which person or organization authenticates that this document is valid.
+  note: If set, must be a Practitioner reference with valid millennium id. Only supported with system access. Cannot be populated for provider access.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator
+  example: |
+    {
+      "authenticator": {
+        "reference": "Practitioner/123456"
+      }
+    }
+
+- name: content
+  required: 'Yes'
+  type: BackboneElement
+  action:
+    - create
+  description: The document and format referenced.
+  note: Must contain only one entry
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.content
+  example: |
+   {
+      "content": [
+        {
+          "attachment": {
+            "contentType": "text/xml;charset=utf-8",
+            "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
+            "title": "title",
+            "creation": "2020-07-29T21:02:04.000Z"
+          }
+        }
+      ]
+   }
+
+- name: content.attachment
+  required: 'Yes'
+  type: Attachment
+  action:
+    - create
+  description: The document or URL of the document along with critical metadata to prove content has integrity.
+  note: Data and content type must be provided. Content Type must include a supported MIME type and character set. The supported MIME Types are `application/pdf`, `text/rtf`, `text/html`, `application/xml`, and `application/xhtml+xml`
+  url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference.content.attachment
+  example: |
+   {
+     "attachment": {
+            "contentType": "application/xml;charset=utf-8",
+            "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
+            "title": "title",
+            "creation": "2020-01-15T14:35:29.000Z"
+          }
+   }
+
+- name: context
+  required: 'Yes'
+  type: BackboneElement
+  action:
+    - create
+  description: The clinical context in which the document was prepared.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context
+  example: |
+   {
+     "encounter": [
+       {
+         "reference": "Encounter/123456"
+       }
+     ],
+     "period": {
+       "start": "2020-01-01T00:00:00.000Z",
+       "end": "2020-01-01T01:00:00.000Z"
+     }
+   }
+
+- name: context.encounter
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Describes the clinical encounter that the document content is associated with.
+  note: Only a single encounter reference is supported.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.context.encounter
+  example: |
+    {
+      "encounter": [
+        {
+          "reference": "Encounter/123456"
```

Comment:
```suggestion
          "reference": "Encounter/4208053"
```

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/1560e66bcece049dcfee1e781105538fe4d7898f